src/viking.js

```diff
@@ -1,14 +1,103 @@
 // Soldier
-class Soldier {}
+class Soldier {
+  constructor(health, strength){
+    this.health = health;
+    this.strength = strength;
+  }
 
-// Viking
-class Viking {}
+  attack(){
+    return this.strength;
+  }
+  receiveDamage(damage){
+    this.health -= damage;
+  }
+}
 
+// Viking
+class Viking extends Soldier {
+  constructor(name, health, strength){
+    super(health, strength);
+    this.name = name;
+    this.health = health;
+    this.strength = strength;
```

Comment:
Aqui você não precisa declarar o `this.health` e o `this.strength` porque ele já está estendendo do Soldier.